backend/src/main/java/com/karankumar/bookproject/backend/model/Book.java

```diff
@@ -244,6 +246,20 @@ public BookBuilder edition(Integer edition) {
         }
     }
 
+    public void addPublisher(@NonNull Publisher publisher) {
+        Set<Publisher> bookPublishers = this.getPublishers();
```

Comment:
We don't need this intermediate variable. We can access the instance variable directly